src/Stratis.Bitcoin/P2P/PeerConnectorConnect.cs

```diff
@@ -63,14 +63,21 @@ public override void OnStartConnect()
         /// </summary>
         public override async Task OnConnectAsync()
         {
+            base.connectedEndpoints = this.ConnectorPeers.Select(x => x.PeerEndPoint).ToArray();
```

Comment:
this is very weird, why would you do that here?

Reply:
Removed connectedEndpoints entirely as per @fassadlr comment.